### PR TITLE
Staff activities

### PR DESCRIPTION
**Pull request by ddxxdd-code**
Implement activities for ServingStaff and DeliverStaff

### CODE REVIEW

File: Android/app/src/main/res/layout/activity_deliverystaff_pick_action.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Chan-Y) [inline Android/app/src/main/res/layout/activity_deliverystaff_pick_action.xml:2]: missing constraints causes warning

File: Android/app/src/release/res/values/google_maps_api.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    you will also need to set up a key for your release certificate.
+
+    Follow the directions here:
+
+    https://developers.google.com/maps/documentation/android/signup
+
+    Once you have your key (it starts with "AIza"), replace the "google_maps_key"
+    string in this file.
+    -->
+    <string name="google_maps_key" translatable="false" templateMergeStrategy="preserve">YOUR_KEY_HERE</string>
```
Comment (Chan-Y) [inline Android/app/src/release/res/values/google_maps_api.xml:19]: missing API_key

**Review by Chan-Y — APPROVED** [review]

File: Android/app/src/release/res/values/google_maps_api.xml (hunk 2 above)
Comment (ddxxdd-code) [reply Android/app/src/release/res/values/google_maps_api.xml:19]: Feature not currently implemented.

File: Android/app/src/debug/res/values/google_maps_api.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    SHA-1 certificate fingerprint:
+    4D:2A:F5:85:FC:F6:8F:B5:1C:FC:85:97:91:8A:5D:EB:3F:8E:29:87
+
+    Alternatively, follow the directions here:
+    https://developers.google.com/maps/documentation/android/start#get-key
+
+    Once you have your key (it starts with "AIza"), replace the "google_maps_key"
+    string in this file.
+    -->
+    <string name="google_maps_key" translatable="false" templateMergeStrategy="preserve">YOUR_KEY_HERE</string>
```
[the remark this answers is not in the document]
Comment (ddxxdd-code) [reply Android/app/src/debug/res/values/google_maps_api.xml:23]: That's true. Currently feature of GoogleMap is not implemented.

**Pull request merged**